### ISSUE TITLE
Cannot parse MTK_BLOADER_INFO_v35

### ISSUE THREAD

**Issue by kenkit** [MofadalAltyeb/MTKPreloaderParser#1]
This file belongs to nokia phone
[preloader.zip](https://github.com/MofadalAltyeb/MTKPreloaderParser/files/7633927/preloader.zip)

**Comment by mfdl** [comment]
Hi, @kenkit , this preloader contains 4 unknown flash id's. is this preloader used in HMD Nokia 2.4 device? any way will take a look later maybe will get the real name of the supported id's.

**Comment by kenkit** [comment]
@MofadalAltyeb Yes it does contain 4 flash id's but it seems that the location of emi's differs from where they normally should be hence why it returns a cid filled with zeroes.
CM2MT2 Also returned the same results so I guess the code is alright.

**Comment by kenkit** [comment]
Maybe this phone uses a different emi structure ?

**Comment by MofadalAltyeb** [comment]
No, the structure is same , but the sub emi contents seems to be encrypted, HMD normally use Foresee.
just noticed that DRAM Rank size is correct and it's the last item on the structure important items,
means all previous data is appended normally.

**Comment by kenkit** [comment]
oooh I thought so actually that file is for an mdm enabled device.
